Event/Subscriber/DoctrineDBALSubscriber.php

```diff
@@ -18,36 +18,36 @@ public static function getSubscribedEvents()
-            'lexik_form_filter.apply.dbal.filter_number'         => array('filterNumber'),
-            'lexik_form_filter.apply.dbal.filter_number_range'   => array('filterNumberRange'),
-            'lexik_form_filter.apply.dbal.filter_text'           => array('filterText'),
+            'lexik_form_filter.apply.dbal.filter_number' => array('filterNumber'),
+            'lexik_form_filter.apply.dbal.filter_number_range' => array('filterNumberRange'),
+            'lexik_form_filter.apply.dbal.filter_text' => array('filterText'),
 
             // Symfony2 field types
-            'lexik_form_filter.apply.dbal.text'     => array('filterText'),
-            'lexik_form_filter.apply.dbal.email'    => array('filterValue'),
-            'lexik_form_filter.apply.dbal.integer'  => array('filterValue'),
-            'lexik_form_filter.apply.dbal.money'    => array('filterValue'),
-            'lexik_form_filter.apply.dbal.number'   => array('filterValue'),
-            'lexik_form_filter.apply.dbal.percent'  => array('filterValue'),
-            'lexik_form_filter.apply.dbal.search'   => array('filterValue'),
-            'lexik_form_filter.apply.dbal.url'      => array('filterValue'),
-            'lexik_form_filter.apply.dbal.choice'   => array('filterValue'),
-            'lexik_form_filter.apply.dbal.country'  => array('filterValue'),
+            'lexik_form_filter.apply.dbal.text' => array('filterText'),
+            'lexik_form_filter.apply.dbal.email' => array('filterValue'),
+            'lexik_form_filter.apply.dbal.integer' => array('filterValue'),
+            'lexik_form_filter.apply.dbal.money' => array('filterValue'),
+            'lexik_form_filter.apply.dbal.number' => array('filterValue'),
+            'lexik_form_filter.apply.dbal.percent' => array('filterValue'),
+            'lexik_form_filter.apply.dbal.search' => array('filterValue'),
+            'lexik_form_filter.apply.dbal.url' => array('filterValue'),
+            'lexik_form_filter.apply.dbal.choice' => array('filterValue'),
+            'lexik_form_filter.apply.dbal.country' => array('filterValue'),
             'lexik_form_filter.apply.dbal.language' => array('filterValue'),
-            'lexik_form_filter.apply.dbal.locale'   => array('filterValue'),
+            'lexik_form_filter.apply.dbal.locale' => array('filterValue'),
             'lexik_form_filter.apply.dbal.timezone' => array('filterValue'),
-            'lexik_form_filter.apply.dbal.date'     => array('filterDate'),
+            'lexik_form_filter.apply.dbal.date' => array('filterDate'),
             'lexik_form_filter.apply.dbal.datetime' => array('filterDate'),
             'lexik_form_filter.apply.dbal.birthday' => array('filterDate'),
             'lexik_form_filter.apply.dbal.checkbox' => array('filterValue'),
-            'lexik_form_filter.apply.dbal.radio'    => array('filterValue'),
+            'lexik_form_filter.apply.dbal.radio' => array('filterValue'),
```

Comment:
@cedric-g it is done automatically by php-cs-fixer, however rather than having this sweeping changes I advise to include a .php_cs such as https://github.com/gushphp/gush/blob/master/.php_cs so that changes are introduced gradually